node_helper.js

```diff
@@ -17,13 +20,15 @@ module.exports = NodeHelper.create({
 	initialize: function (payload) {
 		console.log("initialize");
 		config = payload;
-		var ynabAPI = new ynab.API(config.token);
+		this.ynabAPI = new ynab.API(config.token);
 
 		console.log("created api");
 
-		ynabAPI.budgets.getBudgets().then(budgetsResponse => {
+		this.ynabAPI.budgets.getBudgets().then(budgetsResponse => {
 			console.log("budgetsResponse: " + JSON.stringify(budgetsResponse));
-			ynabBudgetId = budgetsResponse.data.budgets[0].id;
+			//ynabBudgetId = budgetsResponse.data.budgets[0].id;
+			//ynabBudgetId = ynabAPI(config.budget_id);
+			this ynabBudgetId = "22c7f7a8-4924-4b12-97a9-7f198dd9025b";
```

Comment:
Can't leave this here for everyone (not even sure how this doesn't fail).  Unless we make the index a config setting - it should just stay 0 in the master branch.